api/scripts/calculate_percentile_offline.py

```diff
@@ -2,24 +2,30 @@
 import os
 import json
 import pandas as pd
+from pathlib import Path
+from scripts.station_csv_to_json import generate_station_json, load_all_csv_to_dataframe
 
 
-RECENT_YEAR = 2019  # the most recent year that has the core fire season recorded
-RANGES = [(1970, RECENT_YEAR), (RECENT_YEAR - 19, RECENT_YEAR),
+RECENT_YEAR = 2023  # the most recent year that has the core fire season recorded
+RANGES = [(RECENT_YEAR - 29, RECENT_YEAR), (RECENT_YEAR - 19, RECENT_YEAR),
           (RECENT_YEAR - 9, RECENT_YEAR)]
 PERCENTILE = 0.9
 NUMBER_OF_DECIMAL_POINT = 5  # for FWI values
 
+# this is the path to the parent folder containing years of BCWS historical data in csv format
+BASE = Path(R'/path/to/csv/parent/BCWS_datamart_historical_wx_obs')
```

Comment:
Is it possible read the CSV directly from s3?

Also can we folder each datamart dump by year? e.g. `2023/BCWS_datamart_historical_wx_obs` or some way to understand what the latest datadump is next time we run the script.

Reply:
We could also document usage of the script and hold off on this in a follow up task.